tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -802,10 +802,10 @@ abstract class BaseWorksheetLspSuite(
       )
       _ = assertNoDiff(artefactCompletionList, artefactExpectedCompletionList)
 
-      versionExpectedCompletionList = List("0.14.2", "0.14.1", "0.14.0")
+      versionExpectedCompletionList = List("0.13.1", "0.13.0")
       response <- server.completionList(
         "a/src/main/scala/foo/Main.worksheet.sc",
-        "import $dep.`io.circe::circe-yaml:0.14@@`",
+        "import $dep.`io.circe::circe-yaml:0.13@@`",
```

Comment:
This version is not available for scala 3, I think we should use something which works for both scala 2 and 3